src/storage/mvcc/txn.rs

```diff
@@ -324,11 +324,11 @@ impl MvccTxn {
             self.delete_write(key, commit);
             delete_versions += 1;
         }
-        MVCC_VERSIONS_HISTOGRAM.observe(f64::from(versions));
+        MVCC_VERSIONS_HISTOGRAM.observe(versions as f64);
```

Comment:
It is possible to move these observations out of this function now. It makes sense to me to both log and observe in the same place.